_unused_topics/images-using-images-s2i-nodejs-configuration.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * openshift_images/using_images/using-images-source-to-image.adoc
+// * Unused. Can be removed by 4.9 if still unused.
```

Comment:
Unused content. This is brought forward + modularized in case we need it. Since this content is not building in our docs, and will require additional PRs/reviews I think it is safe to skip it for general review.